app/src/main/res/drawable/ic_homemarker.xml

```diff
@@ -0,0 +1,35 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:width="36dp"
```

Comment:
@Dwarikanathmishra  Also Check this height and width of these Drawables. Try to have this specific size